### PR TITLE
updated roundel svg for comments

### PR DESCRIPTION
**Pull request by Joshua-David**
## What does this change?
updated roundel svg for comments

### CODE REVIEW

File: static/src/inline-svgs/icon/marque-36.svg [hunk 1]
```diff
@@ -1,3 +1,6 @@
-<svg width="36" height="36" viewBox="0 0 36 36">
-    <path d="M21.3 8.8c0-4.9-1.5-5.7-3.3-5.7-1.8 0-3.2.7-3.2 5.7s1.5 5.5 3.2 5.5c1.8-.1 3.3-.6 3.3-5.5m-6.5 18.8c-2.3 0-2.9 1.7-2.9 2.9 0 1.8 1.6 3.4 6.3 3.4 5.3 0 6.8-1.5 6.8-3.4 0-1.7-1.3-2.9-3.4-2.9h-6.8zm-4.3-25.2c-6.2 2.8-10.5 9-10.5 16.3 0 4.9 2 9.4 5.2 12.6v-.3c0-3.2 3.1-4.4 5.9-5-2.6-.6-3.9-2.5-3.9-4.4 0-2.6 2.9-4.8 4.3-5.8l-.2-.1c-2.5-1.4-4.1-3.8-4.1-7 0-2.7 1.2-4.9 3.3-6.3m25.5 16.4c0-7.4-4.5-13.8-10.9-16.5 2.1 1.4 3.4 3.5 3.5 6.3l.1.6c0 5.4-4.4 8.2-10.7 8.2-1.6 0-2.7-.1-4.1-.5-.6.4-1.1 1.1-1.1 1.8 0 .9.8 1.6 1.8 1.6h8.8c5.5 0 8.2 2.2 8.2 7.1 0 1.6-.3 3.1-1 4.3 3.3-3.4 5.4-7.9 5.4-12.9"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="0 0 36 36">
```
Comment (sndrs) [inline static/src/inline-svgs/icon/marque-36.svg]: can save a few bytes on this one:

```xml
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36">
    <path fill="#000" d="M18 0a18 18 0 1 0 0 36 18 18 0 0 0 0-36"/>
    <path fill="#FFF" d="M21.2 4.4c2.3.4 5.3 2 6.3 3.1v5.2H27L21.2 5v-.6zm-2.2.4c-4 0-6.3 5.6-6.3 13.2 0 7.7 2.2 13.3 6.3 13.3v.6c-6 .4-14.4-4.2-14-13.8A13.3 13.3 0 0 1 19 4v.7zm10.4 14.4l-1.9.9v8.6c-1 1-3.8 2.6-6.3 3.1V19.9l-2.2-.7v-.6h10.4v.6z"/>
</svg>
```

Comment (Joshua-David) [reply static/src/inline-svgs/icon/marque-36.svg]: whoops forget to optimise these, will do now

File: static/src/inline-svgs/icon/marque-54.svg [hunk 2]
```diff
@@ -1 +1,6 @@
-<svg width="54" height="54" viewBox="0 0 36 36"><path d="M21.3 8.8c0-4.9-1.5-5.7-3.3-5.7-1.8 0-3.2.7-3.2 5.7s1.5 5.5 3.2 5.5c1.8-.1 3.3-.6 3.3-5.5m-6.5 18.8c-2.3 0-2.9 1.7-2.9 2.9 0 1.8 1.6 3.4 6.3 3.4 5.3 0 6.8-1.5 6.8-3.4 0-1.7-1.3-2.9-3.4-2.9h-6.8zm-4.3-25.2c-6.2 2.8-10.5 9-10.5 16.3 0 4.9 2 9.4 5.2 12.6v-.3c0-3.2 3.1-4.4 5.9-5-2.6-.6-3.9-2.5-3.9-4.4 0-2.6 2.9-4.8 4.3-5.8l-.2-.1c-2.5-1.4-4.1-3.8-4.1-7 0-2.7 1.2-4.9 3.3-6.3m25.5 16.4c0-7.4-4.5-13.8-10.9-16.5 2.1 1.4 3.4 3.5 3.5 6.3l.1.6c0 5.4-4.4 8.2-10.7 8.2-1.6 0-2.7-.1-4.1-.5-.6.4-1.1 1.1-1.1 1.8 0 .9.8 1.6 1.8 1.6h8.8c5.5 0 8.2 2.2 8.2 7.1 0 1.6-.3 3.1-1 4.3 3.3-3.4 5.4-7.9 5.4-12.9"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="0 0 36 36">
```
Comment (sndrs) [inline static/src/inline-svgs/icon/marque-54.svg]: should this be 54x54? though not sure why it's different svg tbh?

**Comment by sndrs** [comment]
always running svgs through https://jakearchibald.github.io/svgomg/ to get them as small as possible

File: static/src/inline-svgs/icon/marque-54.svg (hunk 2 above)
Comment (Joshua-David) [reply static/src/inline-svgs/icon/marque-54.svg]: Neither, I just replaced them like for like

**Comment by Joshua-David** [comment]
Ran them through now

File: static/src/inline-svgs/icon/marque-36.svg [hunk 3]
```diff
@@ -1,3 +1 @@
-<svg width="36" height="36" viewBox="0 0 36 36">
-    <path d="M21.3 8.8c0-4.9-1.5-5.7-3.3-5.7-1.8 0-3.2.7-3.2 5.7s1.5 5.5 3.2 5.5c1.8-.1 3.3-.6 3.3-5.5m-6.5 18.8c-2.3 0-2.9 1.7-2.9 2.9 0 1.8 1.6 3.4 6.3 3.4 5.3 0 6.8-1.5 6.8-3.4 0-1.7-1.3-2.9-3.4-2.9h-6.8zm-4.3-25.2c-6.2 2.8-10.5 9-10.5 16.3 0 4.9 2 9.4 5.2 12.6v-.3c0-3.2 3.1-4.4 5.9-5-2.6-.6-3.9-2.5-3.9-4.4 0-2.6 2.9-4.8 4.3-5.8l-.2-.1c-2.5-1.4-4.1-3.8-4.1-7 0-2.7 1.2-4.9 3.3-6.3m25.5 16.4c0-7.4-4.5-13.8-10.9-16.5 2.1 1.4 3.4 3.5 3.5 6.3l.1.6c0 5.4-4.4 8.2-10.7 8.2-1.6 0-2.7-.1-4.1-.5-.6.4-1.1 1.1-1.1 1.8 0 .9.8 1.6 1.8 1.6h8.8c5.5 0 8.2 2.2 8.2 7.1 0 1.6-.3 3.1-1 4.3 3.3-3.4 5.4-7.9 5.4-12.9"/>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="36" height="36"><g fill="none"><path fill="#000" d="M18 0a18 18 0 1 0 0 36 18 18 0 0 0 0-36"/><path fill="#FFF" d="M21.2 4.4c2.3.4 5.3 2 6.3 3.1v5.2H27L21.2 5v-.6zm-2.2.4c-4 0-6.3 5.6-6.3 13.2 0 7.7 2.2 13.3 6.3 13.3v.6c-6 .4-14.4-4.2-14-13.8A13.3 13.3 0 0 1 19 4v.7zm10.4 14.4l-1.9.9v8.6c-1 1-3.8 2.6-6.3 3.1V19.9l-2.2-.7v-.6h10.4v.6z"/></g></svg>
```
Comment (sndrs) [inline static/src/inline-svgs/icon/marque-36.svg]: i'd just confirm you're ok without the viewbox on this one and 54 one – it was present before, i think you might need it? 

also, you can drop the `<g>` element from both i htink

Comment (Joshua-David) [reply static/src/inline-svgs/icon/marque-36.svg]: ok cool

**Comment by Joshua-David** [comment]
Should be all correct now

**Review by sndrs — APPROVED** [review]

**Pull request merged**